front/src/views/news/news.view.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  name: 'ff-news',
+  data: function () {
+    return {
+      newsList: [
+        {
+          id: 1,
+          title: 'Сиквел «Чудо-женщины» перенесли из-за коронавируса',
+          content: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Facilis ea veritat'
+          + 'asasaasdasdadaisajs djams alsas dsad a dad adasasd asd adsada sadasd asdas dasd ad'
+          + 'asasaasdasdadaisajs djams alsas dsad a dad adasasd asd adsada sadasd asdas vlad ad',
+          countViews: 233,
+          previewImage: 'https://localhost:5001/StaticFiles/images/News/1.jfif',
+          publishTime: '18.06.2020 20:30',
+          countCommentaries: 5,
+          author: {
+            id: 1,
+            firstName: 'Мария',
+            lastName: 'Колосова'
+          },
+          categories: [
+            {
+              id: 1,
+              name: 'Variety'
+            },
+            {
+              id: 2,
+              name: 'Film News'
+            }
+          ]
+        }
+      ]
+    };
+  },
+  filters: {
+    modifyCategories (categories) {
```

Comment:
no, modifyCategories is not good one
1. it is filter - it changes presentation of something not to modify object. 
2. and change in what way? do you really understand what does in template filter with this name without js code?
 `{{categories | modifyCategories}}` - don't you think that it obvious that it is something with categories and you still not understand about what change it says?